### PR TITLE
Add instrument breakdown for instruments docs

### PR DESCRIPTION
**Pull request by benfdking**
Type: Docs

- Adds documentation for an instrument breakdown on instruments
- Adds documentation for the difference of instrument and organization
  breakdown
- Adds documentation for use cases of instrument breakdowns

### CODE REVIEW

File: Instrument_Transparency/README.md [hunk 1]
```diff
@@ -1,158 +1,5 @@
...
-                "weight": 0.25
-            },
-        ],
-        "others": 0.125
-    },
-    "readTime": "2020-07-23T09:45:35.311128394Z"
-}
-```
\ No newline at end of file
+For a more direct view of exposure to organizations see the details around [organization breakdown](./OrganizationBreakdown.md), otherwise for a more granular and flexible view see the [instrument breakdown](./OrganizationBreakdown.md).
```
Comment (trelore) [inline Instrument_Transparency/README.md:5]: Both link to organization breakdown, second link should be to instrument breakdown

**Review by trelore — COMMENTED** [review]
Looks good!

File: Instrument_Transparency/InstrumentBreakdown.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+* Must have signed up with Tumelo and obtained the credentials for your service user account (see [Getting Started](../Getting_Started/README.md) for further details)
+* Your habitat must be subscribed to one or more composite instruments
+
+## Definitions
+
+* **Composite instrument**: A composite instrument will be invested into other instruments, such as shares, bonds or even other composite instruments. There are many types of composite instruments (mutual, OEIC, ETFs, etc…) but all have this basic structure.
+
+### Subscribed instruments
+
+When you set up access to Tumelo's API, you tell us which funds you would like to be able to get organization breakdowns for (see Getting Started). We refer to these as your [Subscribed Instruments](https://docs.tumelo.com/#section/Subscribed-Instruments). You may request an organization breakdown on any composite instrument that appears in your subscribed instrument list.
```
[the remark this answers is not in the document]
Comment (wycslingtum) [reply Instrument_Transparency/InstrumentBreakdown.md:22]: it's possible to request an organization breakdown and/or an instrument breakdown for any of the composite instruments that your habitat is subscribed to.

[the remark this answers is not in the document]
Comment (trelore) [reply Instrument_Transparency/InstrumentBreakdown.md:22]: True, however as a new user reading the InstrumentBreakdown doc an organization breakdown might be a surprise.

**Pull request merged**